model/src/main/kotlin/io/spine/validation/required/RequiredIdPatternPolicy.kt

```diff
@@ -37,13 +36,16 @@ import io.spine.server.event.NoReaction
 import io.spine.server.event.React
 import io.spine.server.tuple.EitherOf2
 import io.spine.validation.MessageMarkers
-import io.spine.validation.event.RuleAdded
+import io.spine.validation.event.RequiredFieldDiscovered
 
 /**
- * A policy that marks ID fields in entity state messages and signal messages as required.
+ * A policy that marks ID fields in entity state messages and signal
+ * messages as required.
  *
- * The messages are discovered via the file patterns, specified in [ValidationConfig].
- * If ProtoData runs with no config, this policy never produces any validation rules.
+ * The messages are discovered via the file patterns, specified in
```

Comment:
Which part of the `ValidationConfig` define the patterns? Let's direct the user so that we have less support calls.